app/javascript/blacklight/search_context.js

```diff
@@ -11,34 +11,42 @@ const SearchContext = (() => {
 
   Blacklight.csrfToken = () => document.querySelector('meta[name=csrf-token]')?.content
   Blacklight.csrfParam = () => document.querySelector('meta[name=csrf-param]')?.content
+  Blacklight.searchStorage = () => document.querySelector('meta[name=blacklight-search-storage]')?.content
 
   // this is the Rails.handleMethod with a couple adjustments, described inline:
   // first, we're attaching this directly to the event handler, so we can check for meta-keys
   Blacklight.handleSearchContextMethod = function(event) {
```

Comment:
Is it possible to break this out into two separate functions? This is now 44 lines long and is challenging to make sense of.